test/e2e/framework/jobs_util.go

```diff
@@ -305,6 +307,8 @@ func WaitForJobDeleting(c clientset.Interface, ns, jobName string) error {
 	})
 }
 
+// JobFinishTime returns finish time of the specified job.
+// TODO(oomichi): This is called at a single place, so we can move this later.
```

Comment:
move this to? why does it need moving?

Reply:
The function is used at `test/e2e/node/ttlafterfinished.go` only and I guessed we can move it into the place and make it local function.

Reply:
hmm this seems likely to be useful to something else, I don't think we need this TODO

Reply:
OK, let me remove the TODO.